File: tests/expected/delete.go
package main

import (
	"fmt"
)

func Show() {
	var my_list []int = []int{1, 2, 3, 4, 5}
	my_list = append(my_list[:2], my_list[2+1:]...)
	fmt.Printf("%v\n", len(my_list))
	my_dict := map[string]int{"a": 1, "b": 2, "c": 3}
	delete(my_dict, "b")
	fmt.Printf("%v\n", len(my_dict))
}

func main() {
	Show()
}

File: tests/expected/delete.nim
import tables
proc show() =
  var my_list = @[1, 2, 3, 4, 5]
  my_list.delete(2)
  echo len(my_list)
  var my_dict = {"a": 1, "b": 2, "c": 3}.newTable
  my_dict.del("b")
  echo len(my_dict)

proc main() =
  show()

main()
